docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md

```diff
@@ -335,7 +335,11 @@ options.setCapability("silent", true);
 WebDriver driver = new InternetExplorerDriver(options);
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR to add code sample
+from selenium import webdriver
+
+caps = webdriver.DesiredCapabilities.INTERNETEXPLORER
+caps['silent'] = True
+driver = webdriver.Ie(desired_capabilities=caps)
```

Comment:
Hi @kjayachandra2000 ,

Thanks for the PR.

We're moving away from `desired capabilities`, We need to use driverSpecificOptions to setCapability like described in all other code samples. 

Can you please make the changes accordingly!

Regards,
Harsha